src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseBackupShortTermRetentionPolicy.md

```diff
@@ -0,0 +1,167 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.sql/set-azurermsqldatabasebackupshorttermretentionpolicy
+schema: 2.0.0
+---
+
+# Get-AzureRmSqlDatabaseBackupShortTermRetentionPolicy
+
+## SYNOPSIS
+Gets a backup short term retention policy.
+
+## SYNTAX
+
+### PolicyByInputObject
+```
+Get-AzureRmSqlDatabaseBackupShortTermRetentionPolicy -InputObject <AzureSqlDatabaseModel>
+ [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### PolicyByResourceIdSet
+```
+Get-AzureRmSqlDatabaseBackupShortTermRetentionPolicy -ResourceId <String> [-ResourceGroupName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
```

Comment:
[-ResourceGroupName] <String> is useless here.

Reply:
Updating help docs again.

Reply:
the same for Set-

Reply:
Done